torch/onnx/symbolic_opset9.py

```diff
@@ -2187,16 +2189,29 @@ def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
     numerator = sub(g, input, mean)
     # variance = e((x - e(x))^2), and (x - e(x)) is the numerator in the layer_norm formula
     variance = g.op("ReduceMean", pow(g, numerator, two_cst), axes_i=axes)
-    denominator = sqrt(g, add(g, variance, eps_cst))
+    rdenominator = rsqrt(g, add(g, variance, eps_cst))
```

Comment:
Is this for better performance? I'm slightly concerned if subgraph change like this would lead to backends unable to perform previously pattern based fusion.

Reply:
It's for performance. Your point is also valid so I will replace it with `Sqrt` and `Reciprocal`.